wasp/apps/clock.py

```diff
@@ -19,6 +19,7 @@
 )
 
 MONTH = 'JanFebMarAprMayJunJulAugSepOctNovDec'
+DAY = 'ThuFriSatSunMonTueWed'
```

Comment:
This lookup table doesn't really make any sense. The field that gives the day of the week is numbered 0..6 where 0 is Monday.

Reply:
When I start from Mon... I had a wrong day name displayed. It was Thursday when I did it that's why I start from Tue.
I made changes to now in 
draw.blit(DIGITS[now[*] 
  and see that 
0 is year 
1 is month
2 is day
3 is hour
4 is min
5 not found 
6 is sec
7 and success.

Reply:
I don't understand how you are breaking up the localtime (how can 5 not be found?).

For me:

    >>> watch.rtc.get_localtime()
    (2021, 1, 10, 20, 23, 39, 6, 10)

Where:

* 0 is year (2021)
* 1 month (1)
* 2 is day of the month (10)
* 3 is hour (20)
* 4 is min (23)
* 5 is sec (39)
* 6 is day of the week (6 -> Sunday)
* 7 is day of the year (10 since currently day of the year == day of the month)